src/ObjWriting/ObjContainer/IWD/IWDWriter.cpp

```diff
@@ -0,0 +1,106 @@
+#include "IWDWriter.h"
+
+#include "../minizip/zip.h"
+
+#include <algorithm>
+#include <iostream>
+
+class IWDWriterImpl final : public IWDWriter
+{
+public:
+    explicit IWDWriterImpl(const std::string& iwdFileName, ISearchPath* assetSearchPath)
+        : m_filename(iwdFileName),
+          m_asset_search_path(assetSearchPath),
+          m_files(0)
+    {
+        m_zipfile = zipOpen(m_filename.c_str(), APPEND_STATUS_CREATE);

```

Comment:
As far as i understood it, this does make exactly what you are expecting: It creates the file instead of appending.

The `APPEND_STATUS_` is just the prefix and the `CREATE` is the actual chosen value.

Compared to the source:
```cpp
#define APPEND_STATUS_CREATE        (0)
#define APPEND_STATUS_CREATEAFTER   (1)
#define APPEND_STATUS_ADDINZIP      (2)
```